src/main/java/frc/robot/constants/IntakeConstants.java

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.constants;
+
+/** Add your docs here. */
+public class IntakeConstants {
+    public static final int INTAKE_SPARK_ID = 21; //TODO
```

Comment:
Could you address the TODO comments, or expand them with more context? If you can't address the TODO items right now, then open an issue to track the work that's still needed and leave a link to the issue in the comment here.